### PR TITLE
new my-first-hoodie: the Something Tracker

### PR DESCRIPTION
**Pull request by gr2m**
the original `my-first-hoodie` is [over 3 years old](https://github.com/hoodiehq/my-first-hoodie/commit/609ba5d5e0bb9c71aabc84f3913a68cc05b1b03d). Wow, time flies.

With a new Hoodie release around the corner, it’s time for something new.
### :bulb: Idea: Something Tracker

We would like to build the most simple tracking app imaginable. Track a number with a note, without telling if it’s time, expenses, or whatever. It’s the “Something Tracker”.
### Motivation
- People understand immediately what they can do, there is only one thing to do :)
- Learn like it’s the 90s: right-click, view page source
- when creating a new app, my-first-hoodie gets setup by default. So the app should be great to extend it with own ideas. It will be easy to extend “Something Tracker” to a tracker for expenses, time, inventory
- the app will also be a simple enough to show how to use React, Ember, <you name it> with Hoodie, the adjustments will be easy
- we hope for a ton of hands-on tutorials on how to do the above two things :)
### Features
- responsive design
- add a line with a number and a note
- edit an existing line
- remove an existing line
- sign up / sign in / sign out
### Help :)

The app is only an `index.html`, an empty `app.css` and a few lines in `app.js`. We can use the [store API](https://github.com/hoodiehq/hoodie-client-store#api) as it doesn't need any backend.

We’d love to have some fine people helping us making this great <3
### Preview

The current state can be seen here:

https://rawgit.com/hoodiehq/my-first-hoodie/something-tracker/index.html

### CODE REVIEW

File: assets/app.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* global hoodie */
+
+document.body.addEventListener('submit', function (event) {
```
Comment (NickColley) [inline assets/app.js]: Can this just be on the document? Be good to have comments explaining things like event listeners as I'm not sure super new people would understand what's going on here.

Comment (gr2m) [inline assets/app.js]: I don’t know to be honest, give it a try? And good idea with the comments, wanna add that?

Comment (NickColley) [inline assets/app.js]: Second thought it'd be nicer to add an id to the form and do something like

``` js
document.getElementById('tracker').addEventListener('submit', function () {})
```

I'll add comments If I get the chance.

**Comment by boennemann** [comment]
Could you please make these PRs against the next branch, because that's what is used by hoodie@next without affecting current hoodie apps. Thank you :)

**Comment by gr2m** [comment]
closing in favor of new PR against `next` branch: https://github.com/hoodiehq/my-first-hoodie/pull/85

**Pull request closed without merging**